gms_app/tests.py

```diff
@@ -12,51 +12,65 @@
 def _prepare_db(django_db_setup, django_db_blocker):
     # Set up non-modified objects used by all test methods
     with django_db_blocker.unblock():
-        garden = models.Garden.objects.create(name="Test Garden", description="Test Garden Description")
-        garden_bed = models.GardenBed.objects.create(
-            garden=garden, name="Test Garden Bed", description="Test Garden Bed Description 1"
+        garden = Garden.objects.create(
+            name="Test Garden",
+            description="Test Garden Description",
+        )
+        garden_bed = GardenBed.objects.create(
+            garden=garden,
+            name="Test Garden Bed",
+            description="Test Garden Bed Description 1",
         )
         log.info("Garden '%s' - garden bed '%s'", garden.name, garden_bed.name)
         for plant_i in range(3):
-            p = models.Plant(name=f"Test Plant {plant_i}", description=f"Test Plant Description{plant_i}")
-            p.save()
-            p.garden_bed.add(garden_bed)
-            log.info("\tAdded plant '%s'", p.name)
+            plant = Plant.objects.create(
+                name=f"Test Plant {plant_i}",
+                description=f"Test Plant Description{plant_i}",
+            )
+            planting = Planting.objects.create(
```

Comment:
We could probably update this to use `garden_bed.add_plant(...)`, which is used in the last test

Reply:
Will refactor for next dev push.